swap/src/asb/config.rs

```diff
@@ -151,6 +154,7 @@ pub struct Maker {
     #[serde(with = "::bitcoin::util::amount::serde::as_btc")]
     pub max_buy_btc: bitcoin::Amount,
     pub ask_spread: Decimal,
+    pub kraken_ws_url: Url
```

Comment:
Shall we name this something that doesn't include `kraken`?

My thinking is: there is only one correct `kraken_ws_url`: the one that was previously hardcoded. As such, a config option for that one doesn't make any sense as there is only one valid value.

What we are really configuring here should IMO be named something like `price_ticker_ws_url` and it needs to be documented that we are expecting the Kraken's price ticker protocol on this websocket.

Reply:
https://github.com/comit-network/xmr-btc-swap/pull/585/commits/a222f4c64a4aa16b1adbe9703df58c64925625de